File: Chat/ChatParser/src/MemberDTO.cpp
/* 
 * File:   MemberDTO.cpp
 * Author: dipal
 * 
 * Created on September 11, 2015, 5:53 PM
 */

#include "MemberDTO.h"

namespace imsdk {
MemberDTO::MemberDTO() {
}

MemberDTO::MemberDTO(UserIdentity memberId, UserIdentity ringId, std::string fullName, int status, UserIdentity addedBy) {
    this->memberIdentity = memberId;
    this->ringId = ringId;
    this->fullName = fullName;
    this->status = status;
    this->addedBy = addedBy;
}

MemberDTO::~MemberDTO() {
}

void MemberDTO::setAddedBy(UserIdentity addedBy) {
    this->addedBy = addedBy;
}

UserIdentity MemberDTO::getAddedBy() const {
    return addedBy;
}

void MemberDTO::setStatus(int status) {
    this->status = status;
}

int MemberDTO::getStatus() const {
    return status;
}

void MemberDTO::setFullName(std::string fullName) {
    this->fullName = fullName;
}

std::string MemberDTO::getFullName() const {
    return fullName;
}

void MemberDTO::setMemberIdentity(UserIdentity memberIdentity) {
    this->memberIdentity = memberIdentity;
}

UserIdentity MemberDTO::getMemberIdentity() const {
    return memberIdentity;
}

void MemberDTO::setSeenTime(long long seenTime) {
    this->seenTime = seenTime;
}

long long MemberDTO::getSeenTime() const {
    return seenTime;
}

void MemberDTO::setRingId(UserIdentity ringId) {
    this->ringId = ringId;
}

UserIdentity MemberDTO::getRingId() const {
    return ringId;
}

}

File: Chat/Chat/wrapper/java/JChatSDK/src/com/ringid/imsdk/MemberDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.ringid.imsdk;

/**
 *
 * @author dipal
 */
public class MemberDTO {

    private long memberIdentity;
    private long ringId;
    private String fullName;
    private int status;
    private long addedBy;
    private long seenTime;

    public MemberDTO() {
    }

    public MemberDTO(long memberId, long ringId, String fullName, int status, long addedBy) {
        this.memberIdentity = memberId;
        this.ringId = ringId;
        this.fullName = fullName;
        this.status = status;
        this.addedBy = addedBy;
    }

    public void setAddedBy(long addedBy) {
        this.addedBy = addedBy;
    }

    public long getAddedBy() {
        return this.addedBy;
    }

    public void setStatus(int status) {
        this.status = status;
    }

    public int getStatus() {
        return this.status;
    }

    public void setFullName(String fullName) {
        this.fullName = fullName;
    }

    public String getFullName() {
        return this.fullName;
    }

    public void setMemberIdentity(long memberIdentity) {
        this.memberIdentity = memberIdentity;
    }

    public long getMemberIdentity() {
        return this.memberIdentity;
    }

    /**
     * @return the seenTime
     */
    public long getSeenTime() {
        return seenTime;
    }

    /**
     * @param seenTime the seenTime to set
     */
    public void setSeenTime(long seenTime) {
        this.seenTime = seenTime;
    }

    /**
     * @return the ringId
     */
    public long getRingId() {
        return ringId;
    }

    /**
     * @param ringId the ringId to set
     */
    public void setRingId(long ringId) {
        this.ringId = ringId;
    }
}
